artifacts/src/main/resources/issuance/credential-offer-message-schema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "title": "CredentialOfferMessageSchema",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "#/definitions/CredentialOfferMessage"
+    }
+  ],
+  "$id": "https://w3id.org/dspace-dcp/v0.8/issuance/credential-offer-message-schema.json",
+  "definitions": {
+    "CredentialOfferMessage": {
+      "type": "object",
+      "properties": {
+        "@context": {
+          "$ref": "https://w3id.org/dspace-dcp/v0.8/common/context-schema.json"
+        },
+        "credentialIssuer": {
+          "type": "string"
+        },
+        "credentials": {
+          "type": "array",
+          "items": {
+            "$ref": "https://w3id.org/dspace-dcp/v0.8/issuance/credential-object-schema.json#/definitions/CredentialObject"
+          }
+        }
+      },
+      "oneOf": [
```

Comment:
Is there a particular reason to allow both? I view the alternative between `@type` und `type` as a special type of compaction/expansion which this protocol can choose to restrict.

Reply:
Oh, and for this, you've already opened an issue. 
https://github.com/eclipse-dataspace-dcp/decentralized-claims-protocol/issues/88

Amazing, thanks!

Reply:
Yeah by JSON-LD we can deal with both, but for having a symmetry in the compaction/expansion and examples i would stick with `type` since the credential context and our context does the alias `type` <-> `@type`. But at JSON-LD level they are both valid given that context definition